packages/tldraw/src/lib/shapes/shared/usePrefersReducedMotion.tsx

```diff
@@ -1,10 +1,20 @@
+import { useMaybeEditor, useValue } from '@tldraw/editor'
 import { useEffect, useState } from 'react'
 
 /** @public */
 export function usePrefersReducedMotion() {
+	const editor = useMaybeEditor()
+	const animationSpeed = useValue('animationSpeed', () => editor?.user.getAnimationSpeed(), [
```

Comment:
@MitjaBezensek or @steveruizok I noticed that we don't actually use the user pref for this hook??
any historical reason for that or maybe just has been overlooked thus far?